flink-runtime/src/main/java/org/apache/flink/runtime/throughput/BufferDebloater.java

```diff
@@ -83,6 +81,20 @@ public OptionalInt recalculateBufferSize(long currentThroughput, int buffersInUs
         return OptionalInt.of(newSize);
     }
 
+    @VisibleForTesting
+    boolean skipUpdate(int newSize) {
+        if (newSize == lastBufferSize) {
+            return true;
+        }
+
+        if (newSize <= minBufferSize || newSize >= maxBufferSize) {
```

Comment:
Mostly to double check, we cannot end up here with `newSize < minBufferSize` or `newSize > maxBufferSize`, right? At most we can have equality?

Reply:
Yes, it is true. According to the logic in `BufferDebloater#recalculateBufferSize`, the newSize can not be less than min size and greater than max size.

Reply:
I know it's nitpicky, but can we add a comment stating so? It took me some time to understand, why we are not skipping the update if a `newSize` is below `minBufferSize`.